thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/completeness/checker/DataCompletenessResource.java

```diff
@@ -0,0 +1,58 @@
+package com.linkedin.thirdeye.completeness.checker;
```

Comment:
Yes. If it is to be used for augmenting email alerts, I wonder if it should be part of the dashboard. Wdyt?